docs/demos/coast-to-coast-q4-2023.md

```diff
@@ -0,0 +1,793 @@
+- Overwrite the FBC YAML file to contain the previous content and the new bundle
+```sh
+cat << EOF > catalog/index.yaml
+{
+  "schema": "olm.package",
+  "name": "coastal"
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.0.0-alpha1",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.0.0-alpha1",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.0.0-alpha1"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.0.0",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.0.0",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.0.0"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.1.0",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.1.0",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.1.0"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.0.1",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.0.1",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.0.1"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v2.0.0",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v2.0.0",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "2.0.0"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.channel",
+  "name": "stable",
+  "package": "coastal",
+  "entries": [
+    {
+      "name": "coastal.v2.0.0"
+    },
+    {
+      "name": "coastal.v1.0.1"
+    },
+    {
+      "name": "coastal.v1.1.0"
+    },
+    {
+      "name": "coastal.v1.0.0"
+    },
+    {
+      "name": "coastal.v1.0.0-alpha1"
+    }
+  ]
+}
+EOF
+```
+
+- Build and push the FBC image
+```sh
+docker build -t quay.io/operator-framework/coastal-catalog:latest -f catalog.Dockerfile . && \
+docker push quay.io/operator-framework/coastal-catalog:latest
+```
+
+Once again, we should see the `Catalog` update it's resolved reference and this time we expect that
+nothing happens to the `Operator` resource.
+
+Updating the `Operator`'s version range will still result in a resolution failure since the version installed
```

Comment:
```suggestion
Updating the Operator's version range will still result in a resolution failure since the version installed
```
I would either drop the backticks, or if they are important, use "the `Operator` resource's version range..."

Reply:
[08e8dbb](https://github.com/operator-framework/operator-controller/pull/533/commits/08e8dbba77679d610d04f2b8002613fe94421aff)